client/src/assets/components/subcomponents/Unauthorized.jsx

```diff
@@ -1,13 +1,65 @@
-import React from 'react'
+import React from "react";
+import { Link } from "react-router-dom";
 
 const Unauthorized = () => {
-    return (
-        <div className="text-center mt-10">
-            <h1 className="text-3xl font-bold text-error">403 - Unauthorized</h1>
-            <p className="mt-4">You do not have permission to view this page.</p>
+  return (
+    <div className="fixed inset-0 flex items-center justify-center bg-slate-50 overflow-hidden">
```

Comment:
Can you just update bg-slate-50 to bg-base-300 so it can match the current page backgrounds overall? The same as above. 🙂